src/App.scss

```diff
@@ -55,4 +55,6 @@ $buttonRadius: 10px;
   margin: 0 auto;
   padding: 2rem;
   text-align: center;
+  width: 1280px;
```

Comment:
I think this #root{ } styling is going to be removed, do you rely on a fixed width for the nav bar?